<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ip_port">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>831</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_message">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>350</y>
     <width>451</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
